src/containers/Token/DEXPairs/index.js

```diff
@@ -122,7 +124,7 @@ const DEXPairs = props => {
         </td>
         <td className="issuer-address">
           {pair.issuer !== undefined ? (
-            <a href={`/token/${pair.token}.${pair.issuer}`}>{pair.issuer}</a>
+            <a href={`${urlLink}/token/${pair.token}.${pair.issuer}`}>{pair.issuer}</a>
```

Comment:
Tests were timing out when I changed this to an `InternalLink`, and I couldn't figure out what was wrong, so I left it as an href.